website/content/en/docs/tutorials/iperf.md

```diff
@@ -1,12 +1,228 @@
+    export KUBECONFIG=$PWD/config-peer2
+    ```
+
+Note: You can run the tutorial in a single terminal and switch access between the clusters using  `kubectl config use-context kind-peerX`.
+
+## Deploy iPerf3 Client and Server
+
+1. Install iPerf3 application (Client and server) on the clusters.
+
+    Client cluster:
+
+    ```sh
+    export IPERF3_FILES=https://github.com/clusterlink-net/clusterlink/blob/main/demos/iperf3/testdata/manifests
+    kubectl apply -f $IPERF3_FILES/iperf3-client/iperf3-client.yaml
+    ```
+
+    Server cluster:
+
+    ```sh
+    export IPERF3_FILES=https://github.com/clusterlink-net/clusterlink/blob/main/demos/iperf3/testdata/manifests
+    kubectl apply -f $IPERF3_FILES/iperf3-server/iperf3.yaml
+    ```
+
+## Deploy ClusterLink
+
+1. Create the fabric and peer certificates for the clusters.
+
+    Client cluster:
+
+    ```sh
+    clusterlink create fabric
+    clusterlink create peer-cert --name peer1
+    ```
+
+    Server cluster:
+
+    ```sh
+    clusterlink create peer-cert --name peer2
+    ```
+
+    For more details about fabric and peer concepts see [ClusterLink Concepts](https://github.com/clusterlink-net/clusterlink/docs/concepts/)
+
+1. Deploy ClusterLink on each cluster:
+
+    Client cluster:
+
+    ```sh
+    clusterlink deploy peer --name peer1 --autostart --ingress=NodePort --ingress-port=30443
+    ```
+
+    Server cluster:
+
+    ```sh
+    clusterlink deploy peer --name peer2 --autostart --ingress=NodePort --ingress-port=30443
+    ```
+
+    Note: In this example, we use NodePort to create an external access point for the Kind clusters. By default `deploy peer` creates an ingress of type LoadBalancer, which is more suitable for Kubernetes clusters running in the cloud.
+
+1. Check that ClusterLink controlplane and dataplane are Running.
+    Client cluster:
+
+    ```sh
+    kubectl rollout status deployment cl-controlplane -n clusterlink-system
+    kubectl rollout status deployment cl-dataplane -n clusterlink-system
+    ```
+
+    Server cluster:
+
+    ```sh
+    kubectl rollout status deployment cl-controlplane -n clusterlink-system
+    kubectl rollout status deployment cl-dataplane -n clusterlink-system
+    ```
+
+## Enable Access between the clusters
+
+In this step, we enable connectivity access between the iPerf3 client and server.
+
+1. First, we add the peers to each cluster:
+
+    Client cluster:
+
+    ```sh
+    export PEER2_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' peer2-control-plane`
+    envsubst < $IPERF3_FILES/clusterlink/peer2.yaml | kubectl apply -f -
+    ```
+
+    Server cluster:
+
+    ```sh
+    export PEER1_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' peer1-control-plane`
+    envsubst < $IPERF3_FILES/clusterlink/peer1.yaml | kubectl apply -f -
+    ```
+
+    Note: The `PEER_IP` refers to the node IP of the peer Kind Cluster, which assigns the peer YAML file
```

Comment:
consider, on the entire document, if using the notice/alert etc shortcodes would be helpful.
You can see examples in peers.md file